pom.xml

```diff
@@ -21,11 +21,11 @@
   <parent>
     <groupId>org.jahia.modules</groupId>
     <artifactId>jahia-modules</artifactId>
-    <version>8.0.0.0-SNAPSHOT</version>
+    <version>8.1.1.0</version>
   </parent>
   <groupId>org.jahia.modules</groupId>
   <artifactId>esigate-filter</artifactId>
-  <version>2.0.0-SNAPSHOT</version>
+  <version>2.1.0-SNAPSHOT</version>
```

Comment:
As the groupId of the module is `org.jahia.modules`, You can't change the 2nd digit without updating the signature.
Please change the groupId by a custom one to not require to sign the module